app/controllers/api/v2/auth_source_ldaps_controller.rb

```diff
@@ -46,6 +46,7 @@ def show
           param :tls, :bool
           param :groups_base, String, :desc => N_("groups base DN")
           param :use_netgroups, :bool, :desc => N_("use NIS netgroups instead of posix groups, applicable only when server_type is posix or free_ipa")
+          param :use_rfc4519_group_membership, :bool, :desc => N_("use RFC4519 group membership in addition to posix groups, applicable only when server_type is posix")
```

Comment:
this will require an update to the ansible module too: https://github.com/theforeman/foreman-ansible-modules/blob/ae6b518157a3e099293450f33ead460fd5f195c8/plugins/modules/auth_source_ldap.py#L218